drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -32,8 +32,8 @@ import (
 const (
 	// PXServiceName is the name of the portworx service in kubernetes
 	PXServiceName = "portworx-service"
-	// PXNamespace is the kubernetes namespace in which portworx daemon set runs
-	PXNamespace = "kube-system"
+	//// DefaultPXNamespace is the kubernetes namespace in which portworx daemon set runs
```

Comment:
Is this mean to be commented out? Also nit: DaemonSet is one word

Reply:
yes, forgot to remove it